script.sh

```diff
@@ -99,7 +99,29 @@ else
     gerror
 fi
 
-cp -rH files/"${style}"/Numix/* "${dir}"/Numix/
+read -p "Which folder colour do you want? " answer
+if [[ "default,blue,green,grey,orange,pink,purple,red,yellow" == *$answer* ]]; then
```

Comment:
If the user enter something like "grey,orange", then the script will look for files named "grey,orange*" and maybe crash.
If someone has a better idea to secure this, that would be cool.